File: Algorithms/LinearSearch.py
# Linear Search is basically looking into every element of the array and seeing if the required element is
# present or not

names = ["shehzad","kashan","ahmar","ghufran","maulana sahab"]
found = False
length = len(names)
index = 0

name = input("Enter your name: ")

while not found and index < length - 1:
    if names[index] == name:
        found = True
    else:
        index += 1

if found:
    print("You are registered")
else:
    print("Proceed to counter number 1 for more details. ")

input("Press Enter to continue: ")

# This code is done through in function // Alternate code

#name = input("Enter your name: ")
#if name in names:
#    print("You are already registered. ")
#else:
#    print("Proceed to counter number 1 for more details. ")

File: Algorithms/LinearSearch.vb
Module Module1

    Sub Main()

        Dim names(4) As String
        Dim found As Boolean = False
        Dim index As Integer = 0
        Dim length As Integer
        Dim name As String


        names = {"shehzad", "kashan", "ahmar", "ghufran"}
        length = names.Length 'Len() doesnt work on arrays for some reasons


        Console.WriteLine("Please enter your name: ")
        name = Console.ReadLine()


        While found = False And index < length - 1
            If names(index) = name Then
                found = True
            Else
                index = index + 1
            End If
        End While

        If found Then
            Console.WriteLine("You are already registered")
        Else
            Console.WriteLine("Proceed to counter 1 for more details")
        End If

        Console.ReadKey()

    End Sub

End Module
